safety_limiter/src/safety_limiter.cpp

```diff
@@ -151,13 +154,15 @@ class SafetyLimiterNode
     , has_cloud_(false)
     , has_twist_(true)
     , has_collision_at_now_(false)
+    , stuck_started_since_(ros::Time(0))
   {
     neonavigation_common::compat::checkCompatMode();
     pub_twist_ = neonavigation_common::compat::advertise<geometry_msgs::Twist>(
         nh_, "cmd_vel",
         pnh_, "cmd_vel_out", 1, true);
     pub_cloud_ = nh_.advertise<sensor_msgs::PointCloud>("collision", 1, true);
     pub_debug_ = nh_.advertise<sensor_msgs::PointCloud>("debug", 1, true);
+    pub_status_ = nh_.advertise<safety_limiter_msgs::SafetyLimiterStatus>("status", 1, true);
```

Comment:
`/status` is too general and may cause conflict.

trajectory_tracker is publishing status to the node private namespace.
https://github.com/at-wat/neonavigation/blob/efe62edfc48b57626806984cde1dfd051b4f0972/trajectory_tracker/src/trajectory_tracker.cpp#L187

Reply:
switched to use pnh_ instead.